_posts/2024-05-28-pyos-newsletter-june-2024.md

```diff
@@ -0,0 +1,112 @@
+
+### Tutorials
+#### [3D Visualization with PyVista](https://cfp.scipy.org/2024/talk/GKGRWE/)
+Led by [Tetsuo Koyama](https://cfp.scipy.org/2024/speaker/WY7NA9/), [Alexander Kaszynski](https://cfp.scipy.org/2024/speaker/3RXHG8/), [Bill Little](https://cfp.scipy.org/2024/speaker/FDVWDN/), and [Bane Sullivan](https://cfp.scipy.org/2024/speaker/NEC33M/), this tutorial demonstrates [PyVista's](https://github.com/pyvista/pyvista) latest capabilities and bring a wide range of users to the forefront of 3D visualization in Python, including:
+
+* Use PyVista to create 3D visualizations from a variety of datasets in common formats.
+* Overview the classes and data structures of PyVista with real-world examples.
+* Be familiar of the various filters and features of PyVista.
+* Know which Python libraries are used and can be used by PyVista (meshio, trimesh etc).
+* We see this tutorial catering to anyone who wants to visualize data in any domain, and this ranges from basic Python users to advanced power users.
+
+#### [Create Your First Python Package: Make Your Python Code Easier to Share and Use](https://cfp.scipy.org/2024/talk/QT9GBY/)
+Led by pyOpenSci's Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), in this tutorial session you'll learn how to:
+
+* Create code that can be installed into different environments
+* Use [Hatch](https://hatch.pypa.io/latest/) as a workflow tool, making setup and installation of your code easier
+* Use Hatch to publish your package to (test) PyPI
+
+### Talks
+#### [The power of community in solving scientific Python’s most challenging problems](https://cfp.scipy.org/2024/talk/AMTLJ7/)
+This talk from our Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), shares pyOpenSci’s knowledge in building a constructive open peer review process that supports maintainers, and also resources that make Python packaging more accessible and easier-to-navigate. Our insights are based on five years of experience working with over 200 community members, who have evaluated 58 packages developed by over 56 package maintainers.
+
+#### [Great Tables for Everyone](https://cfp.scipy.org/2024/talk/BRMQRW/)
+This talk from [Rich Iannone](https://cfp.scipy.org/2024/speaker/NJCPZ7/) will provide a demonstration of Great Tables, showing how data can come alive in when in tabular form. We’ll use data from the fields of meteorology, chemistry, and the atmospheric sciences within our table-making examples. We’ll learn how the different components of a table (e.g., the column labels, the header, etc.) can be manipulated to best convey the data. We believe tables are worthy of being deemed data visualizations and we hope that Great Tables can become an indispensable tool in your data analysis workflow.
```

Comment:
this package was submitted for review.

Reply:
yep! also related to isabel - I didn't add intros to these sections, but I will! I think that's a great touch.

Reply:
fantastic!

Reply:
yes!!  i think there are just opportunities to link to say the issue for review (altho there will be a small discussion given data viz tools need to be vetted before review - just a general policy!) ... and yes below isabel is such a wonderful volunteer. wherever we can elevate people's awesomeness let's do it!